gef.py

```diff
@@ -3624,16 +3643,19 @@ def gef_get_auxiliary_values():
     if not is_alive():
         return None
 
-    res = {}
-    for line in gdb.execute("info auxv", to_string=True).splitlines():
-        tmp = line.split()
-        _type = tmp[1]
-        if _type in ("AT_PLATFORM", "AT_EXECFN"):
-            idx = line[:-1].rfind('"') - 1
-            tmp = line[:idx].split()
-
-        res[_type] = int(tmp[-1], base=0)
-    return res
+    try:
+        res = {}
+        for line in gdb.execute("info auxv", to_string=True).splitlines():
+            tmp = line.split()
+            _type = tmp[1]
+            if _type in ("AT_PLATFORM", "AT_EXECFN"):
+                idx = line[:-1].rfind('"') - 1
```

Comment:
What is this finding? I know it's not your code, but if you can figure it out a comment would be nice, or a better var name.

Reply:
I added comments and renamed the variables now. Is that better?